Libraries/Components/Button.windows.js

```diff
@@ -116,6 +147,32 @@ class Button extends React.Component {
     );
     const formattedTitle = Platform.OS === 'android' ? title.toUpperCase() : title;
     const Touchable = Platform.OS === 'android' ? TouchableNativeFeedback : TouchableOpacity;
+    let content =
+        <View style={buttonStyles}>
+          <Text style={textStyles} disabled={disabled}>{formattedTitle}</Text>
+        </View>;
+
+    if (Platform.OS === "windows") {
+      const tabIndex = this.props.tabIndex || 0;
+      const windowsTabFocusable = !disabled && tabIndex >= 0;
+      content =
+        <FocusableWindows
+          ref = {(ref) => this._setFocusableRef(ref)}
+          disabled={disabled}
+          isTabStop={windowsTabFocusable}
+          tabIndex={tabIndex}
+          disableSystemFocusVisuals={this.props.disableSystemFocusVisuals}
+          handledKeyDownKeys={DOWN_KEYCODES}
+          handledKeyUpKeys={UP_KEYCODES}
+          onKeyDown={this._onKeyDown}
+          onKeyUp={this._onKeyUp}
+          onFocus={this._onFocus}
+          onBlur={this._onBlur}
+        >
+          {content}
+      </FocusableWindows> ;
```

Comment:
>  [](start = 25, length = 1)

extra space

Reply:
fixing

---
In reply to: [157097872](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r157097872) [](ancestors = 157097872)